Modules/Sources/NetworkingCore/Validators/PlaceholderDataValidator.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+/// Placeholder implementation for `ResponseDataValidator`.
+///
+public final class PlaceholderDataValidator: ResponseDataValidator {
+    public init() {}
+
+    public func validate(data: Data) throws {
+        // no-op
+    }
+}
```

Comment:
Again, just a diff that Git/GitHub can't resolve as rename + edit. Previous version:

https://github.com/woocommerce/woocommerce-ios/blob/8089b161d25c39ddb4bff8ab12342f474c67f882/Networking/Sources/Core/Validators/PlaceholderDataValidator.swift#L1-L9